library/src/main/java/com/otaliastudios/zoom/ZoomImageView.java

```diff
@@ -89,12 +90,50 @@ public boolean onTouchEvent(MotionEvent ev) {
     public void onUpdate(ZoomEngine helper, Matrix matrix) {
         mMatrix.set(matrix);
         setImageMatrix(mMatrix);
+
+        if (!awakenScrollBars()) {
+            invalidate();
```

Comment:
Any reason for this call? We should not need to invalidate. setImageMatrix will do it.
Same for ZoomLayout.

Reply:
The documentation for ```awakenScrollBars()``` states:

```
When the animation is started, this method returns true, and false otherwise. If the animation is started, this method calls {@link #invalidate()}; in that case the caller should not call {@link 
 invalidate()}.
```

So I added this condition. I guess if ```setImageMatrix``` already does it this check is useless.

Reply:
Yeah I think it's better to simply call awakenScrollBars() and ignore the result.
Also, you need to merge the other PR changes to merge this one. Thanks!